functions/functions.go

```diff
@@ -114,6 +114,9 @@ func (f *Functions) DeleteFunction(id FunctionID) error {
 	if err != nil {
 		return &ErrorNotFound{id}
 	}
+
+	f.Log.Info("function deleted", zap.String("functionId", string(id)))
```

Comment:
@mthenw For the message (`Function deleted`) or for the name of the key in the key-value pair? This tracks the other logs